internal/gitstream/assign.go

```diff
@@ -13,10 +13,10 @@ import (
 	gh "github.com/qbarrand/gitstream/internal/github"
 	"github.com/qbarrand/gitstream/internal/gitutils"
 	"github.com/qbarrand/gitstream/internal/markup"
-	"gopkg.in/yaml.v3"
+	"github.com/qbarrand/gitstream/internal/owners"
 )
 
-const ownersFile = "OWNERS"
+const ownersFile = "../../OWNERS"
```

Comment:
I think we should get the downstream config to build the path to that path.

Reply:
Fixed.